app/index.html

```diff
@@ -81,11 +81,16 @@
             </h1>
           </div>
 
+          <div class="card-pf-heading">
+            <h2 class="card-pf-title">
+              Recommender
+            </h2>
+          </div>
```

Comment:
Leave them as two separate cards for now, and @mindreeper2420 can sort out the UX.